netlify.toml

```diff
@@ -11,3 +11,15 @@
 
 # [build.environment] # not sure if we'll need to specify a version
   # RUBY_VERSION = "some.version.of.ruby"
+
+[[redirects]]
+  from = "http://melta.no/*"
+  to = "http://docs.meltano.com/guide/troubleshooting/:splat"
```

Comment:
could just redirect directly to https and cut down the # of redirects that would happen.

Reply:
Good point. We already redirect to HTTPS for other domains for the docs site.